src/components/new-request/analyze-results.vue

```diff
@@ -82,12 +82,13 @@
                 <v-col cols="auto">
                   <div v-for="(corp, n) in conflicts" :key="'conflict-' + n">
                     <b>{{ corp.name }}</b><br>
-                    <span v-if="conflictDate && conflictId.startsWith('NR ')">
-                      <b>Submitted Date:</b> {{ conflictDate + ', ' }}</span>
-                    <span v-if="conflictDate && !conflictId.startsWith('NR ')">
-                      <b>Incorporation Date:</b> {{ conflictDate + ', ' }}</span>
-                    <span v-if="conflictId && !conflictId.startsWith('NR ')">
-                      <b>Corporation No.:</b> {{ conflictId }}</span>
+                    <template v-if="conflictId.startsWith('NR ')">
+                      <span><b>Submitted Date:</b> {{ conflictDate }}</span>
+                    </template>
+                    <template v-if="!conflictId.startsWith('NR ')">
+                      <span><b>Incorporation Date:</b> {{ conflictDate + ', ' }}</span>
+                      <span><b>Corporation No.:</b> {{ conflictId }}</span>
+                    </template>
```

Comment:
The old code did not display the line when the value was null.
The new code always displays the line (with "Unknown" value if it's null). This is the concept of "letting the user know there's something there, even if we can't say what it is" (ie, don't hide error).